language/types_values_variables.md

```diff
@@ -142,16 +146,17 @@ may appear more than once in the hierarchy (e.g. a `Scalar` is both `Any` and `D
        |  |     |- (Integer with range inside another Integer)
        |  |  |- Float[from, to]
        |  |     |- (Float with range inside another Float)
+       |  |  |- Timespan
+       |  |  |- Timestamp
        |  |
        |  |- String[from, to]
        |  |  |- Enum[*strings]
        |  |  |- Pattern[*patterns]
        |  |
        |  |- Boolean
        |  |- Regexp[pattern_string]
-       |
-       |- SemVer
-       |- SemVerRange
+       |  |- SemVer
+       |  |- SemVerRange
```

Comment:
This is problematic because Scalar is considered Data IIRC. We then end up with a translation problem as SemVer and SemVerRange cannot be represented as Data without using some kind of mapping. (That was the reason they were not added to the table as Scalars even if they are scalars in practice). It makes more sense to fix Scalar in Data then not having SemVer types be Scalar though.